src/content/en/updates/2017/02/media-session.md

```diff
@@ -0,0 +1,398 @@
+          { src: 'https://dummyimage.com/128x128', sizes: '128x128', type: 'image/png' },
+          { src: 'https://dummyimage.com/192x192', sizes: '192x192', type: 'image/png' },
+          { src: 'https://dummyimage.com/256x256', sizes: '256x256', type: 'image/png' },
+          { src: 'https://dummyimage.com/384x384', sizes: '384x384', type: 'image/png' },
+          { src: 'https://dummyimage.com/512x512', sizes: '512x512', type: 'image/png' },
+        ]
+      });
+
+      navigator.mediaSession.setActionHandler('play', function() {});
+      navigator.mediaSession.setActionHandler('pause', function() {});
+      navigator.mediaSession.setActionHandler('seekbackward', function() {});
+      navigator.mediaSession.setActionHandler('seekforward', function() {});
+      navigator.mediaSession.setActionHandler('previoustrack', function() {});
+      navigator.mediaSession.setActionHandler('nexttrack', function() {});
+    }
+
+## Get into the code
+
+### Let's play 🎷
+
+Add a simple `<audio>` tag to your web page and assign several media sources so
+that the browser can choose which one works best.
+
+    <audio controls>
+      <source src="audio.mp3" type="audio/mp3"/>
+      <source src="audio.ogg" type="audio/ogg"/>
+    </audio>
+
+Note: I could have used a `<video>` tag as well there to showcase the Media
+Session API.
+
+As you may know, `autoplay` is disabled on Chrome for Android which
+means we have to use the `play()` method of the audio element there. This
+method must be triggered by [a user gesture] such as a touch or a mouse click.
+We're talking about listening to
+[`pointerup`](/web/updates/2016/10/pointer-events), `click`, and `touchend`
+events. In other words, the user must click on a button before your web app can
+actually make some noise.
+
+    playButton.addEventListener('pointerup', function(event) {
+      let audio = document.querySelector('audio');
+
+      // User interacted with the page. Let's play audio...
+      audio.play()
+      .then(_ => { /* Set up Media Session... */ })
+      .catch(error => { console.log(error) });
+    });
+
+If you don't want to play audio right after the first interaction, I recommend
+you use the `load()` method of the audio element. This is one way for the
+browser to keep track of whether the user interacted with the element. Note
+that it may also help the playback to be smoother because the content will be
+loaded.
+
+<pre class="prettyprint">
+
+let audio = document.querySelector('audio');
+
+welcomeButton.addEventListener('pointerup', function(event) {
+  // User interacted with the page. Let's load audio...
+  <strong>audio.load()</strong>
+  .then(_ => { /* Show play button for instance... */ })
+  .catch(error => { console.log(error) });
+});
+
+// Later...
+playButton.addEventListener('pointerup', function(event) {
+  <strong>audio.play()</strong>
+  .then(_ => { /* Set up Media Session... */ })
+  .catch(error => { console.log(error) });
+});
+
+</pre>
```

Comment:
Is this section necessary? The audio element has `controls`, could we just rely on that? Then we can skip all the user interaction and "pointerup" stuff.

That stuff's important, but feels like it could be a different article.

In fact, does the code in this article fail if the user interacts with the audio element controls rather than `playButton`?